composer.json

```diff
@@ -19,15 +19,15 @@
     "prefer-stable": true,
     "require": {
         "php": ">=5.3.9",
-        "symfony/framework-bundle": "~2.3.27 || ^2.6.6 || ~3.0",
-        "symfony/doctrine-bridge": "~2.3 || ~3.0",
+        "symfony/framework-bundle": "~2.7 || ~3.0",
+        "symfony/doctrine-bridge": "~2.7 || ~3.0",
```

Comment:
@lsmith77 do we have an issue with this? i guess we are too far with 1.3 to include this PR in this release, so this would affect 1.4 only...